src/components/api.js

```diff
@@ -0,0 +1,96 @@
+const Api = {
+
+  // Fetches buildings data from the API
+
+  searchData: async (json) => {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(json)
+    }
+    try {
+      const response = await fetch('https://api.urban-brussels.osoc.be/search', options)
+      const json = await response.json()
+      return json
+    } catch (err) {
+      console.log('Error getting documents', err)
+    }
+  },
+
+  getAutocomplete: async (lang, query) => {
+    const options = {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }
+    try {
+      const response = await fetch('https://api.urban-brussels.osoc.be/autocomplete?lang=' + lang + '&query=' + query, options)
+      const json = await response.json()
+      return json
+    } catch (err) {
+      console.log('Error getting documents', err)
```

Comment:
`console.log()` is not enough: the user will not check the console but still should be warned if there is an error.